calibration_with_blender/calibration_benchmark.cpp

```diff
@@ -0,0 +1,684 @@
+#include "opencv2/core.hpp"
+#include <opencv2/core/utility.hpp>
+#include "opencv2/imgproc.hpp"
+#include "opencv2/3d.hpp"
```

Comment:
I propose to have `#ifdef` with OpenCV version here to have proper includes according to OpenCV version. I do not see 5.x specific code here.